assets/src/dashboard/app/views/savedTemplates/header/index.js

```diff
@@ -46,6 +46,22 @@ function Header({ filter, search, sort, templates, view }) {
     actions: { scrollToTop },
   } = useLayoutContext();
 
+  const searchOptions = useMemo(() => {
```

Comment:
Nit: should we pull this out into a helper function since it's used in a couple places?

Tbh it's very readable so I'm not worried about repeating code. Adding an abstraction here doesn't win us anything.

Reply:
we could, i feel like it won't always be the same. savedTemplates is mostly just a shadow of my stories right now in an attempt to keep it "current"  so that when it's time to revisit and resurrect it, it isn't tragically out of date. That said, a helper doesn't hurt!